docs/en/advanced/Performance-Tuning.md

```diff
@@ -166,71 +170,67 @@ consider reducing this value so that checkpoints happen more often.
-connection pool can be drained, preventing new connections from being established.
+The `alluxio.worker.block.threads.max` property configures the maximum number of incoming RPC requests to 
+worker that can be handled. 
+This value is used to configure maximum number of threads in Thrift thread pool of the block worker.
+This value should be greater than the sum of `alluxio.user.block.worker.client.threads` across concurrent Alluxio clients.
+Otherwise, the worker connection pool can be drained, preventing new connections from being established.
 
-### Async caching
+### Async block caching
 
-Since Alluxio 1.7.0 improves the performance of cold-reading a block partially or non-sequentially
-with the default read type. Previously clients used a flag to force full reads of blocks in order to
-store them into Alluxio to speed up follow up reads on the same blocks. Now these reads for caching
-purpose will be handled by the Alluxio in the background, vastly decreasing the latency of partial
-read requests in many workloads. For example, reading the first 10MB of a 512MB block with partial
-caching on required a read of the entire block (512MB); now the client reads 10MB and continues
-processing while an Alluxio worker loads the 512MB block in the background.
+When a worker requests for data from a portion of a block, the worker reads as much data as requested
+and immediately returns the requested data to the client.
+The worker will asynchronously continue to read the remainder of the block without blocking the client request.
 
-A tunable configuration for this feature is the
-`alluxio.worker.network.netty.async.cache.manager.threads.max` property. This determines the maximum
-number of concurrent blocks a worker will attempt to cache. By default the value is high (512), but
-in cases where large amounts of data are expected to be asynchronously cached concurrently, lowering
-the value can reduce resource contention (e.g., # of cores)
+The number of asynchronous threads used to finish reading partial blocks is set by the
+`alluxio.worker.network.netty.async.cache.manager.threads.max` property, with a default value of `512`.
+When large amounts of data are expected to be asynchronously cached concurrently, it may be helpful
+to reduce this value to reduce resource contention.
 
 ### Netty Configs
 
-`alluxio.worker.network.netty.rpc.threads.max` - Default value is 2048, consider increasing this value
-if you have a very large number of concurrent client accesses and are seeing threadpool exhausted
-errors on the worker.
+The number of RPC threads available on a worker is configured by
+`alluxio.worker.network.netty.rpc.threads.max`, with a default value of `2048`.
+This value should be increased if exhausted thread pool errors are being logged on workers.
 
 ### Async eviction and watermarks
+TODO: this is the exact same topic as "Evicting Stale Data" in Alluxio Storage Management. only keep one???
```

Comment:
??

Reply:
this is actually the default behavior now. I will remove this session